### DISCUSSION TITLE
.url 文件点击跳转到网页支持

### DISCUSSION THREAD

**Discussion by OrzMiku** [AlistGo/alist#4004] [Ideas]
`.url`文件是Internet快捷方式

baidu.url
```
[InternetShortcut]
URL=https://www.baidu.com
```

这样的文件在电脑上点开就可以直接跳转到`https://www.baidu.com`

将`.url`文件上传到alist，点击也能实现同样的效果

**Comment by TheStoneFish** [comment]
> `.url`文件是Internet快捷方式
> 
> baidu.url
> ```
> [InternetShortcut]
> URL=https://www.baidu.com
> ```
> 
> 这样的文件在电脑上点开就可以直接跳转到`https://www.baidu.com`
> 
> 将`.url`文件上传到alist，点击也能实现同样的效果
可以用md 吧 url应该不行

**Reply by OrzMiku** [reply]
md需要点到文件里面，然后再点链接。和url不一样，url就在文件列表中，点击就可以直接跳转。
希望能做一下对url格式的支持。
